.gitignore

```diff
@@ -19,6 +19,14 @@ test/benchmark/baseline.txt
 xcuserdata/
 *.xccheckout
 
+# Allow Visual Studio project files.
+!wren/
+!wren_lib/
+
+# Visual Studio intermediate files.
+Debug/
+Release/
```

Comment:
Would you be OK with tweaking the VS project to put these under `build/debug` and `build/release`? That matches the other platforms and makes the docs that talk about build outputs correct across Mac/Linux/Windows.